server/formatters/kebabCase.ts

```diff
@@ -0,0 +1,6 @@
+export default function kebabCase(str: string) {
```

Comment:
you have added helper classes , formatters and validators. Do we need similar things in the other FE projects and if so, would it make sense to have a them in a separate project and include them as a dependency?

Reply:
Generally we don't do this as it add complexity to build, involves versioning libraries and generally creates over-head.  The chances are other project will not use it.